src/test/scala/org/broadinstitute/dsde/firecloud/webservice/WorkspaceApiServiceSpec.scala

```diff
@@ -1109,21 +1103,20 @@ class WorkspaceApiServiceSpec extends BaseServiceSpec with WorkspaceApiService w
 
     "WorkspaceService importPFB Tests" - {
 
-      "should 400 if import service indicates a bad request" in {
-
+      "should bubble up 400 from cwds" in {
         (Post(pfbImportPath, PFBImportRequest("https://bad.request.avro"))
           ~> dummyUserIdHeaders(dummyUserId)
           ~> sealRoute(workspaceRoutes)) ~> check {
-            status should equal(BadRequest)
-            responseAs[String] should include ("Bad request as reported by import service")
+            status should equal(InternalServerError) // TODO: bubble up 400 / BadRequest
```

Comment:
The tests verify _current_ behavior in prod, however these tests used to verify what happened in the interaction to `ImportService` so I left these TODOs in place to trigger a conversation over whether we should continue to strive to resurface errors, or if we're okay with just bubbling up as `InternalServerError`.  If we're okay with leaving this as is, I could get rid of the TODOs.

Reply:
My vote would be to deprecate Orchestration's import related endpoints entirely and move everything to talk directly to CWDS.

Reply:
For these APIs, I would much rather bubble up the actual error/response code instead of wrapping everything in a 500. Since this current PR does not change behavior, I'm fine making that change in a follow-on.

Reply:
>For these APIs, I would much rather bubble up the actual error/response code instead of wrapping everything in a 500. Since this current PR does not change behavior, I'm fine making that change in a follow-on.

I plan to leave out any changes related to error handling from this PR, but wanted to highlight @nawatts 's suggestion here which is worth considering: https://github.com/broadinstitute/firecloud-orchestration/pull/1355#discussion_r1598481587